include/nlohmann/detail/input/parser.hpp

```diff
@@ -144,8 +143,10 @@ class parser
         return sax_parse(&sax_acceptor, strict);
     }
 
-    bool sax_parse(json_sax_t* sax, const bool strict = true)
+    template <typename SAX>
+    bool sax_parse(SAX* sax, const bool strict = true)
     {
+        // (void)detail::is_sax_static_asserts<SAX, BasicJsonType>{};
```

Comment:
Either the line should be removed or it should be un-commented.

Reply:
Oops, this should be un-commented